oemoflex/preprocessing.py

```diff
@@ -1283,7 +1283,11 @@ def normalize_year(timeseries):
         profile_file_suffix = settings['profile-file-suffix']
         profile_name_suffix = settings['profile-name-suffix']
 
-    mapping_filepath = os.path.join(module_path, 'flexmex_config', 'mapping-input-timeseries.yml')
+    path_config = os.path.join(module_path, 'model_config', 'experiment_paths.yml')
+    with open(path_config, 'r') as config_file:
+        config_path = yaml.safe_load(config_file)['config']
+
+    mapping_filepath = os.path.join(module_path, config_path, 'mapping-input-timeseries.yml')
```

Comment:
This looks weird. `config_path` replaces the relative path `flexmex_config`. For that, another relative path `model_config` is introduced. It gets more complicated. Still, I prefer this over `flexmex_config` because `model_config/experiment_paths.yml` is module intern. What do you think?